.github/workflows/ci-pr.yml

```diff
@@ -16,4 +16,6 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
       - name: Build package
-        run: mvn install
\ No newline at end of file
+        run: mvn install
+      - name: Test package
```

Comment:
I believe this is not necessary because mvn runs them at install

Reply:
yes, thats right